core/templates/dev/head/domain/question/QuestionBackendApiService.ts

```diff
@@ -52,6 +59,35 @@ oppia.factory('QuestionPlayerBackendApiService', [
       });
     };
 
+    var _fetchQuestionSummaries = function(
+        skillIds, cursor, successCallback, errorCallback) {
+      if (!isListOfStrings(skillIds)) {
+        errorCallback('Skill ids should be a list of strings');
+        return false;
+      }
+      var questionsDataUrl = UrlInterpolationService.interpolateUrl(
+        QUESTIONS_LIST_URL_TEMPLATE, {
+          skill_ids: skillIds.join(','),
```

Comment:
key name should be comma_separated_skill_ids

Reply:
Done.